src/prune/tooling/ui.cpp

```
@@ -65,12 +62,7 @@ namespace prune {
 
                 if (!m_file_status.empty()) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)